src/masonite/foundation/Application.py

```diff
@@ -48,9 +49,16 @@ def get_providers(self):
     def __call__(self, *args, **kwargs):
         return self.response_handler(*args, **kwargs)
 
+    def is_debug(self):
+        """Check if debug mode is enabled."""
+        return bool(config("application.debug"))
```

Comment:
The problem is this won't work for existing applications. The new 4.0 apps we didn't add this config variable in the config file

Reply:
It will work. At least, it won't raise an error. this will return False.
So all existing projects won't be in debug mode until they add the line in `config/application.py`.

So it's still a problem I agree, we can until next major release, use:
```python
    def is_debug(self):
        """Check if debug mode is enabled."""
        if Config.has("application.debug"):
             return bool(config("application.debug"))
        else:
             return env("APP_DEBUG")   # using env() here to ensure that it's converted to bool
```

Reply:
That will work yeah